TLM/TLM/State/OptionsTabs/MaintenanceTab_ConfigGroup.cs

```diff
@@ -13,10 +13,15 @@ public static class MaintenanceTab_ConfigGroup {
             Handler = OnReloadGlobalConfigClicked,
         };
         public static ActionButton ResetGlobalConfig = new() {
-            Label = "Maintenance.Button:Reset global configuration",
+            Label = "Maintenance.Button:Reset all configuration",
```

Comment:
? This will be super confusing for users. 
1, they will ask what happened to reset global config button
2. they will ash what does "all" even mean.

Reply:
global is also not accurate because we are resetting defaults for new game too. 
@krzychu124 what do you suggest I do?

Reply:
I'm not sure why you decided to save defaults in global config?
Global settings are user regardless of state but the defaults would be used only in specific condition which suggest that is not the best place for that.
Maybe we just need something like TMPE_DefaultGameSettings.xml created automatically, and then the user could even change things by hand? This way we could easily handle multiple versions of settings, e.g.: with popup with list of configs to use, similar to MoveIt Import/Export modal. That seem to be more future proof and most likely easier to manage.

Reply:
or maybe saved game options can have different text color in main menu.

@krzychu124 but I still don't understand what should I call that button? should I split it into 2 buttons?

Reply:
Leave it like it was "Global configuration" has its own meaning in the mod, wiki etc., always refers to GlobalConfig configuration file